### DISCUSSION TITLE
登陆被腾讯拦截怎么办

### DISCUSSION THREAD

**Discussion by Kosue02** [Arondight/Adachi-BOT#19] [Q&A] [answered]
[禁止登录]当前上网环境异常，请更换网络环境或在常用设备上登录或稍后再试。(错误码：237)

**Comment by Arondight** [comment]
如果用户名密码都正确的话，这种情况我不知道……没有遇到过，你可以去 [oicq](https://github.com/takayama-lily/oicq) 问问有没有解决方案。

**Comment by Arondight** [comment]
我刚注册的小号也被拦截了，你是换 IP 还是等了几天自动好的？

**Reply by Kosue02** [reply]
https://github.com/takayama-lily/oicq/wiki/02.%E5%85%B6%E4%BB%96%E5%B8%B8%E8%A7%81%E9%97%AE%E9%A2%98
下这个到自己手机上，然后用真机划和获取ticket，这样就不会拦截了

**Reply by Arondight** [reply]
尴尬，这个我看了，但是没有安卓手机，我想办法借一个吧

**Reply by mark9804** [reply]
> 这个我看了，但是没有安卓手机

随便下一个模拟器就可以，哪怕打游戏用的mumu也行

**若你使用模拟器，则需要打开桥接模式，否则依然无法得到正确的ticket**

https://github.com/takayama-lily/oicq/wiki/02.%E5%85%B6%E4%BB%96%E5%B8%B8%E8%A7%81%E9%97%AE%E9%A2%98

**Comment by FireGuo** [comment]
我拿滑动验证助手还是会提示上网环境异常该怎么解决呢

**Comment by mark9804** [comment, marked as answer]
若ticket输入正确仍然环境异常，可考虑使用扫码登录。或安装：https://github.com/mzdluo123/TxCaptchaHelper/releases 中的apk，在安卓真机上滑动后得到ticket

若你使用模拟器，则需要打开桥接模式，否则依然无法得到正确的ticket

我之前遇到过一次，过了一会儿再试就成功了

参考：
* https://github.com/takayama-lily/oicq/wiki/01.%E6%BB%91%E5%8A%A8%E9%AA%8C%E8%AF%81%E7%A0%81%E5%92%8C%E8%AE%BE%E5%A4%87%E9%94%81
* https://github.com/takayama-lily/oicq/wiki/02.%E5%85%B6%E4%BB%96%E5%B8%B8%E8%A7%81%E9%97%AE%E9%A2%98
